src/Gir/Marshal/SymbolTable.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Gir
+{
+	public partial class SymbolTable : IEnumerable<ISymbol>
+	{
+		// FIXME: Known issue with pointers. We need to figure that one out.
```

Comment:
Just remove any leading const and trailing *? But I don't think that we should do that in the symbol table but in the ISymbols itself

Reply:
i.e. `gint8*` is not registered in the dictionary.

Reply:
But does that matter? gint8 should be registered and that resolves to a primitive which then can implement IPassByReference? And if gint8* is an array we can detect that in from CType.Array.

Reply:
Ah, I was just commenting on my FIXME. Definitely, that is the way to go.